Tests/SwiftLintFrameworkTests/PrefixedTopLevelConstantRuleTests.swift

```diff
@@ -0,0 +1,27 @@
+@testable import SwiftLintFramework
+import XCTest
+
+final class PrefixedTopLevelConstantRuleTests: XCTestCase {
+    func testDefaultConfiguration() {
+        verifyRule(PrefixedTopLevelConstantRule.description)
+    }
+
+    func testPrivateOnly() {
+        let triggeringExamples = [
+            "private let ↓Foo = 20.0",
+            "fileprivate let ↓foo = 20.0"
+        ]
+        let nonTriggeringExamples = [
+            "let Foo = 20.0",
+            "internal let Foo = \"Foo\"",
+            "public let Foo = 20.0"
+        ]
+
+        let alwaysOnSameLineDescription = PrefixedTopLevelConstantRule.description
```

Comment:
can you rename this variable? (I just noticed it's also wrong on the linked PR 😅 )

Reply:
Ah yea, fixed